internal/test/smoke/system_endpoints_test.go

```diff
@@ -87,20 +95,60 @@ func (s *SuiteSystemEndpoints) TestHostConfExecute() {
 	t := s.T()
 	s.As(RBACSuperAdmin)
 	s.prepareDomainIpa(t)
-
-	t.Log("Calling SystemHostConfWithResponse")
 	domainType := public.RhelIdm
 	s.As(XRHIDSystem, RBACNoPermis)
-	res, err := s.HostConfWithResponse(
-		s.domain.RhelIdm.Servers[0].SubscriptionManagerId.String(),
-		"client."+s.domain.DomainName,
-		builder_api.NewHostConf().
-			WithDomainName(pointy.String(s.domain.DomainName)).
-			WithDomainType(&domainType).
-			Build())
-	require.NoError(t, err)
-	require.NotNil(t, res)
-	require.Equal(t, http.StatusOK, res.StatusCode)
+
+	t.Run("HostConf success", func(t *testing.T) {
+		// Expect success HostConf event to be sent to Pendo
+		mockPendo, ok := s.PendoClient.(*mock_pendo.Pendo)
+		require.True(t, ok)
+		mockPendo.On("SendTrackEvent", mock.Anything, mock.MatchedBy(func(r *pendo.TrackRequest) bool {
+			return (r.Event == "HostConf" &&
```

Comment:
I suggest to use `PendoHostConfSuccess` constant; that would keep the test in sync with the event value. I suggest to move to a common place the constant (thinking on `internal/interface/client/pendo` package, but I recognize not 100% happy with that suggested location as it would be providing service dependent values at a place which is client independent place).

Reply:
Do we have some high level package where similar stuff can belong. E.g. `internal.domain.pendo` and we can have all track events on single place.

Reply:
:+1: good idea! that is a nice place land that values and share them cross the application.